CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 
+* [ENHANCEMENT] Ability to toggle off latency or count metrics in metrics-generator [#2070](https://github.com/grafana/tempo/pull/2070) (@AlexDHoffer)
```

Comment:
Why not include size as well? Either as part of `count` or its own type.

Reply:
Mostly was interested in getting a copy up to verify the approach made sense to maintainers before I finished polishing